### ISSUE TITLE
Profile backgrounds as in the good days of pre-2015 tt

### ISSUE THREAD

**Issue by danimesq** [bluesky-social/social-app#5088]
**Is your feature request related to a problem? Please describe.**

Yes, the problem that social networks are losing their fun since 2014 (when Brazillians lost Orkut).

**Describe the solution you'd like**

As users can upload the profile pic and profile cover, they could upload a profile background.

**Describe alternatives you've considered**

None.

**Additional context**

On Settings, users could opt-out of this feature or, if disabled by default, opt-in (adding a background to your profile should mean automatically opting-in).

**Comment by Sininini** [comment]
Yeah! Bsky should have more features that set it apart from other modern social media
